routes/index.js

```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```

Comment:
accounting relies on this param being description and not a flag. I'm not sure how that process works, though. Feels busted to me.